### PR TITLE
Document location of GUT panel in editor

### PR DESCRIPTION
**Pull request by nycdotnet**
I am new to Godot but an experienced unit tester. It took me a bit to figure out where the GUT panel was.  Hopefully this helps to clear it up for the next person.  Thanks for creating this.

### CODE REVIEW

File: documentation/docs/Quick-Start.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Setup doesn't get any simpler than what is already on the [install page.](Instal
 
 
 ## Run Tests
-* Configure the directories where your tests are in the Gut Panel settings.
+* Configure the directories where your tests are in the [Gut Panel](Install#activate) settings.
```
Comment (bitwes) [inline documentation/docs/Quick-Start.md:10]: Change this to:
```
* Open the GUT panel
![Gut Panel Location](_static/images/gut_panel_where.png)
* Configure the directories where your tests are in the [Gut Panel](Install#activate) settings.
```

**Review by bitwes — COMMENTED** [review]
Thank you!  I keep meaning to add this and then not getting to it.  I've requested a minor change to include the image you added in the Quick Start page.

**Comment by nycdotnet** [comment]
OK I think I have it good now.  Note the extra whitespace is required to get the GitHub flavored markdown to render with the wide image on the line below.  It looks ok on my fork where the image links.

https://github.com/nycdotnet/Gut/blob/docs-gut-panel/documentation/docs/Quick-Start.md

**Comment by nycdotnet** [comment]
OK I think this seems pretty good now, but please let me know if you want any other changes (or for example if I should clean up the screenshots and hand-drawn arrow)

**Review by bitwes — CHANGES_REQUESTED** [review]
I like the additional image for configuring directories, but I thought it should have a little more context.  So I started typing and moving things around...and then typed some more.  I came up with the following which should replace the "Setup" section (now Install) and the "Run Tests" section.

```
## Install
Install GUT from the Asset Library.  For more details and alternate install methods, check the [install page](Install).


## Running Tests

### Open the GUT panel
Click the word GUT in the bottom bar of the editor.<br>
![GUT Panel Location](_static/images/gut_panel_where.png)

GUT Panel Parts<br>
![Gut Panel](_static/images/gut_panel.png)


### Configure Directories
* In the Settings Subpanel, configure the directories where your tests are/will be (you may need to scroll the Settings Subpanel).
* "Include Subdirs" will include any subdirectory of any directories you enter (on by default).

![GUT Panel Test Directories](_static/images/gut_panel_test_directories.png)

### Run Your Tests
* Click "Run All" to run all your tests (top left of the GUT Panel).
* Run a single script:  open a test script and click the button with your test script's name (after "Current", test_test.gd in the GUT Panel image).
* Run an Inner Class:  You will get a button for an InnerClass the cursor is in as well (TestAssertEq in the GUT Panel image)
* Run a single test:  put the cursor inside a test function, click the button with your test function's name (test_fails_when_number_not_equal in the GUT Panel image).

Mouse-over labels and buttons in the GUT panel for more information.  You can even set keyboard shortcuts for all of the GUT panel actions ("Configure Shortcuts" in the GUT Panel image).

You can also run tests via the [command line](Command-Line) and through [VSCode](https://marketplace.visualstudio.com/items?itemName=bitwes.gut-extension)
```

Feel free to make adjustments.  This made sense to me, but the help isn't for me.

If want to make the arrows look more similar to the arrows in the GUT Panel image I would appreciate that.

**Comment by bitwes** [comment]
Do you think the "Creating Tests" section should come before the "Running Tests" section?  I kept wanting to either make reference to it in "Running Tests", which then made me want to move it.

**Comment by nycdotnet** [comment]
Yeah probably.  Will take another cut.

**Comment by nycdotnet** [comment]
OK how's this? It's mostly your copy but I made some tweaks and also made two example unit tests just for demonstration's sake.

**Comment by nycdotnet** [comment]
I didn't update the images - if you're happy with this I can still do that if you want.

**Review by bitwes — APPROVED** [review]
Looks good.  Thank you.  As I said this one has been on the list for awhile, I'm really happy to have this done.

**Comment by nycdotnet** [comment]
cool!

**Pull request merged**